packages/twenty-front/src/modules/workflow/components/WorkflowRunOutputVisualizer.tsx

```diff
@@ -0,0 +1,29 @@
+import { useWorkflowRun } from '@/workflow/hooks/useWorkflowRun';
+import styled from '@emotion/styled';
+import { isDefined } from 'twenty-ui';
+
+const StyledSourceCode = styled.pre`
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  border: ${({ theme }) => `1px solid ${theme.border.color.medium}`};
+  background-color: ${({ theme }) => theme.background.transparent.secondary};
+  margin: ${({ theme }) => theme.spacing(4)};
+  padding: ${({ theme }) => theme.spacing(2)};
+  white-space: pre-wrap;
+`;
+
+export const WorkflowRunOutputVisualizer = ({
+  workflowRunId,
+}: {
+  workflowRunId: string;
+}) => {
+  const workflowRun = useWorkflowRun({ workflowRunId });
+  if (!isDefined(workflowRun)) {
+    return null;
+  }
+
+  return (
+    <StyledSourceCode>
```

Comment:
I think you could use the <CodeEditor component here. See an example in readonly CodeEditor usage in `packages/twenty-front/src/modules/settings/serverless-functions/components/tabs/SettingsServerlessFunctionTestTab.tsx` line 99
```suggestion
         <CodeEditor
            files={[
              {
                content: workflowRun.output,
                language: 'json',
                path: 'result.any',
              },
            ]}
            currentFilePath={'result.any'}
            options={{ readOnly: true, domReadOnly: true }}
          />
```

Reply:
Great idea! I made the change. It looks like that, now:

![CleanShot 2024-10-17 at 14 00 15@2x](https://github.com/user-attachments/assets/e8eff434-84e7-4a4f-929b-5e3d3641013d)